pages/_document.jsx

```diff
@@ -0,0 +1,24 @@
+import {
+  Html, Head, Main, NextScript,
+} from 'next/document';
+
+export default function Document() {
+  return (
+    <Html lang="en">
+      <Head>
+        <link href="/favicon/favicon.ico" rel="shortcut icon" />
```

Comment:
`favicon.ico` doesn't exist, so we can delete this line.

Reply:
I can still delete it if it's not needed but `favicon.ico` does exist in the favicon folder. In the commit I did delete the boilerplate `favicon.ico` that existed in the root of the public directory and the file changes displays the new one with `Binary file not shown.` so that might be the source of some confusion.

Reply:
Ah! I see it now – the icon was different in my editor, which apparently meant my brain ignored it completely. We should be all set, then!